configs/diem_developer_website.json

```diff
@@ -0,0 +1,32 @@
+{
+  "index_name": "diem_developer_website",
+  "start_urls": [
+    "https://developers.diem.com/",
+    "https://developers.diem.com/docs/welcome-to-diem"
+  ],
+  "sitemap_urls": [
+    "https://developers.diem.com/sitemap.xml"
+  ],
+  "sitemap_alternate_links": true,
+  "stop_urls": [],
+  "selectors": {
+    "lvl0": {
+      "selector": ".menu__link--sublist.menu__link--active",
+      "global": true,
+      "default_value": "Documentation"
+    },
+    "lvl1": "[class^='docItemContainer_'] h1",
+    "lvl2": "[class^='docItemContainer_'] h2",
+    "lvl3": "[class^='docItemContainer_'] h3",
+    "lvl4": "[class^='docItemContainer_'] h4",
+    "lvl5": "[class^='docItemContainer_'] h5",
+    "text": "[class^='docItemContainer_'] p, [class^='docItemContainer_'] li"
+  },
+  "selectors_exclude": [
+    ".hash-link"
+  ],
+  "conversation_id": [
+    "1176378229"
+  ],
+  "nb_hits": 989
+}
```

Comment:
I use the same values here as another config - https://github.com/algolia/docsearch-configs/blob/master/configs/libra-website.json

I hope that is ok?

Reply:
All good, I updated it to make sure it match your website structure.